Marlin/stepper.cpp

```diff
@@ -655,7 +655,7 @@ ISR(TIMER1_COMPA_vect) {
     // Take multiple steps per interrupt (For high speed moves)
     for (int8_t i = 0; i < step_loops; i++) {
       #ifndef USBCON
-        MSerial.checkRx(); // Check for serial chars.
+        MarlinCustomizedSerial.checkRx(); // Check for serial chars.
```

Comment:
As with the others here, `MarlinCustomizedSerial` is an instance and not a class. Instance names should start with a lowercase letter.